tests/test-vip-mail.php

```diff
@@ -6,6 +6,21 @@ public function setUp() {
 		reset_phpmailer_instance();
 	}
 
+	public function test__disable_vip_smtp() {
+		$GLOBALS['all_smtp_servers'] = [ 'server1', 'server2' ];
+
+		add_filter( 'vip_smtp_enabled', '__return_false' );
```

Comment:
This runs too late for the test to actually work. Need to work to run before `init` somehow. Or re-initialize the VIP_SMTP class.

Reply:
Can we just do another `( new VIP_SMTP() )->init()` after that filter was set?

Reply:
Ok tested that out and it didn't work